src/polus/plugins/_plugins/classes/plugin_classes.py

```diff
@@ -373,13 +377,40 @@ def submit_plugin(
 
     # Save the manifest if it doesn't already exist in the database
     organization = plugin.containerId.split("/")[0]
-    org_path = PLUGIN_DIR.joinpath(organization.lower())
+    org_path = _PLUGIN_DIR.joinpath(organization.lower())
     org_path.mkdir(exist_ok=True, parents=True)
     if not org_path.joinpath(out_name).exists():
         with open(org_path.joinpath(out_name), "w") as fw:
-            json.dump(plugin.dict(), fw, indent=4)
+            m = plugin.dict()
+            m["version"] = m["version"]["version"]
+            json.dump(m, fw, indent=4)
 
-    # Refresh plugins list if refresh = True
-    if refresh:
-        _r()
+    # Refresh plugins list
+    refresh()
     return plugin
+
+
+def remove_plugin(plugin: str, version: typing.Optional[str] = None):
+    """Remove plugin from the local database."""
+    if not version:
```

Comment:
You should check explicitly for `if version is not None` because the empty string also evaluates to `False`.